src/amo/components/SearchResult.js

```diff
@@ -22,6 +22,9 @@ class SearchResult extends React.Component {
           <section className="SearchResult-main">
             <img className="SearchResult-icon" src={result.icon_url} alt="" />
             <h2 className="SearchResult-heading">{result.name}</h2>
+            <h3 className="SearchResult-rating">
+              <span className="visually-hidden">4/5</span>
```

Comment:
This is totally just a placeholder (and should be labelled as such) while we implement review components.
So it should be translated, but for now is it cool if I just add a TODO that says "replace this when real reviews come"?